### PR TITLE
[MODULAR]Blueshield semi-rework

### PR DESCRIPTION
**Pull request by Vizoplez**
## About The Pull Request
Adds the elite M45A5, a modular gun that supports both 10mil rounds and a custom subtype, the 460, furthermore, also switches their krav-maga for guerilla gloves - powerful tackling implements that are also insulated
image
image

image

## Why It's Good For The Game
Massively changes blueshield to be once more defensive, giving them more options past running up and abusing krav maga.
Also allows them to dive infront of the captain to eat bullets, or to tackle targets without too much bullshit.

## Changelog
:cl:
add: M45A5 pistol, 460R ammo type
del: Blueshields krav maga gloves
add: Guerilla gloves to blueshield equipment
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/blueshield/code/modules/projectiles/guns/ballistic/ballistic.dm [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+	icon = 'modular_skyrat/modules/blueshield/icons/obj/guns/M45A5.dmi'
+	icon_state = "rowlandmodular"
+	ammo_type = /obj/item/ammo_casing/b10mm
+	caliber = "10mm"
+	max_ammo = 15 //Tactical mags.
+	multiple_sprites = AMMO_BOX_FULL_EMPTY_BASIC
+
+/obj/item/ammo_box/magazine/multi_sprite/ladon/blueshield/b460
+	name = "ACA 460-10 modular magazine .460"
+	desc = "A hand-assembled custom sporting handgun by Alpha Centauri Armories, chambered in .460 Rowland Magnum. Loaded with 460 by default."
```
Comment (Draganfrukts) [inline modular_skyrat/modules/blueshield/code/modules/projectiles/guns/ballistic/ballistic.dm]: Why's this one got a description if none of the others don't?

Comment (Vizoplez) [reply modular_skyrat/modules/blueshield/code/modules/projectiles/guns/ballistic/ballistic.dm]: Its to differentiate between mags loaded with 10mil and the ones loaded with 460, 460 is a unique ammo that /cannot/ be found anywhere else except these three mags. Its good to distinguish incase the blueshield swaps out two mags for 10mils. The 460 is VERY lethal.

Comment (Vizoplez) [reply modular_skyrat/modules/blueshield/code/modules/projectiles/guns/ballistic/ballistic.dm]: now defunct because I can make defines work

**Comment by Vizoplez** [comment]
Now ready for merge, all sprites and mechanics work

File: modular_skyrat/master_files/code/defines/_ammo_defines.dm [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#define AMMO_TYPE_LETHAL		"lethal"
+#define AMMO_TYPE_RUBBER		"rubber"
+#define AMMO_TYPE_HOLLOWPOINT	"hollowpoint"
+#define AMMO_TYPE_IHDF 			"ihdf"
+#define AMMO_TYPE_460 			".460"
```
Comment (Gandalf2k15) [inline modular_skyrat/master_files/code/defines/_ammo_defines.dm]: You don't really have to define this here, seems abit redundant, just put round_type = ".460" to save a pointless define.

Comment (Vizoplez) [reply modular_skyrat/master_files/code/defines/_ammo_defines.dm]: fair

File: modular_skyrat/modules/blueshield/code/modules/jobs/job_types/blueshield.dm [hunk 3]
```diff
@@ -29,12 +29,12 @@
...
 	suit = /obj/item/clothing/suit/armor/vest/blueshield
-	suit_store = /obj/item/gun/ballistic/automatic/pistol/m1911
-	gloves = /obj/item/clothing/gloves/krav_maga/combatglovesplus
+	suit_store = /obj/item/gun/ballistic/automatic/pistol/blueshield
+	gloves = /obj/item/clothing/gloves/tackler/combat/insulated
 	shoes = /obj/item/clothing/shoes/jackboots
 	ears = /obj/item/radio/headset/heads/blueshield/alt
 	glasses = /obj/item/clothing/glasses/hud/security/sunglasses
-	backpack_contents = list(/obj/item/ammo_box/magazine/m45 = 3,/obj/item/melee/baton/blueshieldprod = 1)
+	backpack_contents = list(/obj/item/ammo_box/magazine/multi_sprite/ladon/blueshield/b460 = 3,/obj/item/melee/baton/blueshieldprod = 1)
```
Comment (Gandalf2k15) [inline modular_skyrat/modules/blueshield/code/modules/jobs/job_types/blueshield.dm]: Haha you know what I'm gonna say here don't you? Make it a gunkit :3

Comment (Vizoplez) [reply modular_skyrat/modules/blueshield/code/modules/jobs/job_types/blueshield.dm]: Gunkits are for armadyne stuff :^(

Comment (Vizoplez) [reply modular_skyrat/modules/blueshield/code/modules/jobs/job_types/blueshield.dm]: but alas, sure

File: modular_skyrat/modules/blueshield/code/modules/projectiles/guns/ballistic/ballistic.dm [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+	round_type = AMMO_TYPE_RUBBER
+
+/obj/item/ammo_box/magazine/multi_sprite/ladon/blueshield/ihdf
+	ammo_type = /obj/item/ammo_casing/b10mm/ihdf
+	round_type = AMMO_TYPE_IHDF
+
+/obj/item/storage/box/gunset/blueshield
+	name = "ACA E-M45A5 Gunset"
+	w_class = WEIGHT_CLASS_NORMAL
+
```
Comment (Gandalf2k15) [inline modular_skyrat/modules/blueshield/code/modules/projectiles/guns/ballistic/ballistic.dm:67]:
```suggestion
/obj/item/gun/ballistic/automatic/pistol/blueshield/nomag
	spawnwithmagazine = FALSE

```

File: modular_skyrat/modules/blueshield/code/modules/projectiles/guns/ballistic/ballistic.dm [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+/obj/item/storage/box/gunset/pdh_captain/PopulateContents()
+	. = ..()
+	new /obj/item/gun/ballistic/automatic/pistol/blueshield(src)
```
Comment (Gandalf2k15) [inline modular_skyrat/modules/blueshield/code/modules/projectiles/guns/ballistic/ballistic.dm]:
```suggestion
	new /obj/item/gun/ballistic/automatic/pistol/blueshield/nomag(src)
```

**Review by Gandalf2k15 — CHANGES_REQUESTED** [review]

**Review by Gandalf2k15 — APPROVED** [review]
We're gonna just send it lads

**Comment by Gandalf2k15** [comment]
SEND IT TO THE MOON

**Pull request merged**